include/itkImageToTorchImageFilter.h

```diff
@@ -19,60 +19,88 @@
 #define itkImageToTorchImageFilter_h
 
 #include "itkImageToImageFilter.h"
+#include "itkImageRegionIterator.h"
+#include "itkImageRegionConstIterator.h"
+
+#include "itkTorchImage.h"
 
 namespace itk
 {
 
 /** \class ImageToTorchImageFilter
  *
- * \brief Filters a image by iterating over its pixels.
+ * \brief Converts an itk::Image to an itk::TorchImage.
  *
- * Filters a image by iterating over its pixels in a multi-threaded way
- * and {to be completed by the developer}.
+ * Conversions between Image and TorchImage permit pipelines with
+ * steps for either kind of image.
  *
  * \ingroup PyTorch
  *
  */
-template <typename TInputImage, typename TOutputImage>
-class ImageToTorchImageFilter : public ImageToImageFilter<TInputImage, TOutputImage>
+template< typename TInputImage, typename TOutputTorchImage >
```

Comment:
The second parameter might not be needed. Even if kept, it should be defaulted to expected definition, e.g.:
```suggestion
template< typename TInputImage, typename TOutputTorchImage = TorchImage< typename TInputImage::PixelType, TInputImage::ImageDimension > >
```

Reply:
I will omit the second parameter.